### DISCUSSION TITLE
Calibre as font, change width and alignment index page

### DISCUSSION THREAD

**Discussion by freekvh** [gohugo-ananke/ananke#762] [2 - Q&A / Support]
Hi, this is more of a support question than an issue, I hope this is the right place for this. I have several (noobish) questions.

1. How would I change the color of the Title/Name (the large text on top of image on the index page)?
2. A client want to use Calibre as a font, setting body_classes = "calibre bg-near-white" works, but is this recommended? Since calibre is not in the supported fonts listed [here](https://github.com/tachyons-css/tachyons/blob/v4.7.0/src/_font-family.css). Btw, an alternative would be helvetica neue but how do you add a font with a space in the name?
3. How can I change the width and alignment of the text on the index page?

If this is not the right place to ask all this, please let me know where I could go with these questions, thank you! And thanx for this great theme!

**Comment by regisphilibert** [comment]
I guess the Discussions would be ideal to ask for support. The theme uses tachyons so any classes from their documentation should work: https://tachyons.io/#style
